tests/requirements-libraries.txt

```diff
@@ -22,16 +22,12 @@ pylint==1.7.2
 markdown==2.6.9
 simplejson==3.11.1 # simplejson is used for text_c_extension
 
-# sphinx 1.5 requires Python 2.7 or >= 3.4
-sphinx==1.6.3; python_version != '3.3'
-sphinx==1.4.9; python_version == '3.3' # pyup: ignore
+sphinx==1.6.3
 
```

Comment:
Please also remove the empty lines. There is no use in keeping them, now the "block" is gone. Same below.